query-node/mappings/src/content/video.ts

```diff
@@ -163,12 +163,7 @@ export async function content_VideoUpdated(ctx: EventContext & StoreContext): Pr
     await processVideoMetadata(ctx, video, newMetadata, videoUpdateParameters.assets_to_upload.unwrapOr(undefined))
   }
 
-  // create nft
-  const issuanceParameters = videoUpdateParameters.auto_issue_nft.unwrapOr(null)
-  const nft = issuanceParameters ? await createNft(store, video, issuanceParameters, event.blockNumber) : undefined
-
-  // update the video
-  video.nft = nft
```

Comment:
Yes, it seems that there was some problem when rebasing the code on top of the newest rhodes. I'll look into it

Reply:
The fixture file also got removed tests/network-tests/src/fixtures/content/nft/utils.ts

Reply:
The feature has been recovered. I've also noticed and fixed a small bug there -> in case `auto_issue_nft` is unset, video NFT will get unset even when it already exists.